README.md

```diff
@@ -1,11 +1,21 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="https://github.com/keploy/keploy/tree/main/src/images/logo.png" width="30%" />
+</p>
+
+[![Slack](https://img.shields.io/badge/slack-@keyploy--slack-yellow.svg)](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA)
+[![Github](https://img.shields.io/badge/github-@keyploy--github-9cf.svg)](https://github.com/keploy)
```

Comment:
Since this is the main Keploy Repository, it is not required to mention the keploy org profile here.

Reply:
Remove